src/fhirtypes/ElementDefinition.ts

```diff
@@ -1626,17 +1635,17 @@ export class ElementDefinition {
     });
   }
   /**
-   * Finds and returns all fixable descendents of the element. A fixable descendent is a direct child of the
-   * element that has minimum cardinality greater than 0, and all fixable descendents of such children
-   * @returns {ElementDefinition[]} the fixable descendents of this element
+   * Finds and returns all assignable descendents of the element. A assignable descendent is a direct child of the
+   * element that has minimum cardinality greater than 0, and all assignable descendents of such children
+   * @returns {ElementDefinition[]} the assignable descendents of this element
    */
-  getFixableDescendents(): ElementDefinition[] {
-    const fixableChildren = this.children(true).filter(e => e.min > 0);
-    let fixableDescendents: ElementDefinition[] = [];
-    fixableChildren.forEach(fc => {
-      fixableDescendents = fixableDescendents.concat(fc.getFixableDescendents());
+  getAssignableDescendents(): ElementDefinition[] {
+    const assignableChildren = this.children(true).filter(e => e.min > 0);
+    let assignableDescendents: ElementDefinition[] = [];
+    assignableChildren.forEach(fc => {
```

Comment:
This is incredibly minor, but the fc variable here should probably be ac instead now. But I'm thinking this might be too nit picky and it's just fine. I'm guessing you agree since that's what you have now. Is that a fair assumption?

Reply:
You're giving me a lot of credit to assume that I noticed the `fc` vs `ac`. I will change this, and do a quick look for some other sneaky `f`s.

Reply:
I hadn't noticed any others, so I'd say the others are fair game to stay at this point.